src/deferred.js

```diff
@@ -0,0 +1,13 @@
+
+var oldDeferred = jQuery.Deferred;
+
+jQuery.Deferred = function() {
+	var deferred = oldDeferred.apply( this, arguments );
+
+	deferred.pipe = function() {
```

Comment:
So this will add the `pipe` method to the deferred, but not to the promise - using `.then` or `.promise` would yield a promise that does not have a `pipe` method unless I'm mistaken.

In order to get the method available everywhere it needs to be, this probably needs to look like

```js
function pipe() { ... }

deferred.pipe = deferred.promise().pipe = pipe;
```